.github/workflows/build_and_test.yml

```diff
@@ -49,6 +49,10 @@ on:
         required: false
         type: string
         default: ''
+    secrets:
+      codecov_token:
+        description: The upload token of codecov.
+        required: false
```

Comment:
So the previous issue was whether `codecov_token` shouldn't specify `default`?

Reply:
`type` and `default` cannot be set in `secrets`, otherwise an error will be reported.